packages/auth/e2e/auth.e2e.js

```diff
@@ -15,7 +15,41 @@
  *
  */
 
+const TEST_EMAIL = 'test@test.com';
+const TEST_PASS = 'test1234';
+
+const DISABLED_EMAIL = 'disabled@account.com';
+const DISABLED_PASS = 'test1234';
+
+const { clearAllUsers } = require('./helpers');
+
 describe('auth()', () => {
+  before(async () => {
+    try {
+      await clearAllUsers();
+      await firebase.auth().createUserWithEmailAndPassword(TEST_EMAIL, TEST_PASS);
+    } catch (e) {
+      // they may already exist, that's fine
+    }
+    // Need a way to create the user but have it disabled immediately
+    // If we had the functions emulator running as well, it would work:
+    // https://stackoverflow.com/questions/43140441/how-to-disable-user-accounts-on-firebase-authentication-automatically#comment73363725_43141780
+    // try {
+    //   const userCredential = await firebase
+    //     .auth()
+    //     .createUserWithEmailAndPassword(DISABLED_EMAIL, DISABLED_PASS);
+    // } catch (e) {
+    //   // they may already exist, that's fine
+    // }
```

Comment:
Can hold off for now and hopefully in future something like this becomes available on the emulator 🤞